pandas/io/parsers/python_parser.py

```diff
@@ -1135,16 +1136,26 @@ class FixedWidthReader(abc.Iterator):
     A reader of fixed-width lines.
     """
 
-    def __init__(self, f, colspecs, delimiter, comment, skiprows=None, infer_nrows=100):
+    def __init__(
+        self,
+        f: IO[str],
+        colspecs: list[tuple[int, int]] | str,
+        delimiter: str | None,
+        comment: str | None,
+        skiprows: set[int] | None = None,
```

Comment:
Could probably also be a callable: https://github.com/pandas-dev/pandas/blob/8149da0b16a666a34b44306eaaa9b6ea301aa5ed/pandas/io/parsers/readers.py#L1126

Reply:
No callable raises